nixos/modules/services/matrix/appservice-discord.nix

```diff
@@ -114,7 +121,7 @@ in {
 
       preStart = ''
         if [ ! -f '${registrationFile}' ]; then
-          ${pkgs.matrix-appservice-discord}/bin/matrix-appservice-discord \
+          ${config.package}/bin/matrix-appservice-discord \
```

Comment:
This and other references should use `cfg` instead of `config` to refer to this module's configuration.